frontend/src/components/Notification/Notification.tsx

```diff
@@ -62,15 +62,15 @@ const Notification = () => {
       </div>
     ));
 
-  const element = document.getElementsByClassName(
-    'notification_bell__HML54'
-  )[0]!;
-  element.removeAttribute('aria-describedby');
+  useEffect(() => {
+    const element = document.getElementById('popup');
+    element?.removeAttribute('aria-describedby');
+  }, []);
 
   return (
     <Popup
       trigger={
-        <button className={styles.bell}>
+        <button id="popup" className={styles.bell}>
```

Comment:
id's should be unique, but there's no guarantee for this since it's sitting in a component. Maybe use the `useId` hook to give this element a unique id?